netutils/ip.py

```diff
@@ -596,3 +597,37 @@ def get_usable_range(ip_network: str) -> str:
         lower_bound = str(net[1])
         upper_bound = str(net[-2])
     return f"{lower_bound} - {upper_bound}"
+
+
+def sort_list_ips(ips: t.Union[str, t.List[str]], sort_type: str = "network") -> str:
+    """Given a concatenated list of CIDRs sorts them into the correct order and returns as concatenated string.
```

Comment:
Can you throw an example in the docstring?

Reply:
Done!

Reply:
Sorry, I should've specified. Something like 

```python
Examples:
        >>> from netutils.ip import sort_list_ips
        >>> sort_list_ips("3.3.3.3,2.2.2.2,1.1.1.1")
        1.1.1.1,2.2.2.2,3.3.3.3
```

The docs pull it in so it needs to be in the same format as the others.

Reply:
Oops, my bad! I should have noticed the others. Should be fixed now.